pkg/osx/req.txt

```diff
@@ -18,6 +18,7 @@ psutil==5.6.6
 pyasn1==0.4.8
 pycparser==2.19
 pycryptodomex==3.9.7
+pyobjc==6.2
```

Comment:
This needs to be as it was, though the version can/should be upgraded and the `.txt` compiled again.
The reason being that static requirements compilation doesn't not handle pyobjc because it tries to inspect the system at install/build time.

Reply:
okay, can do.

Reply:
Thank You!